agent/handlers/task_server_setup_test.go

```diff
@@ -3561,3 +3566,114 @@ func TestUpdateTaskProtection(t *testing.T) {
 		},
 	}))
 }
+
+func TestRegisterHandler(t *testing.T) {
```

Comment:
TestRegisterFaultHandler

Reply:
Ah yeah that's fair, will address in a follow up PR. thanks